compiler/src/iree/compiler/Codegen/LLVMGPU/TransformExtensions/LLVMGPUExtensions.cpp

```diff
@@ -655,5 +656,33 @@ transform_dialect::VectorToMMAConversionOp::applyToOne(
   return DiagnosedSilenceableFailure::success();
 }
 
+DiagnosedSilenceableFailure transform_dialect::PromoteOperandsOp::applyToOne(
+    Operation *target, transform::ApplyToEachResultList &results,
+    transform::TransformState &state) {
+  OpBuilder builder(getContext());
+  OpBuilder::InsertionGuard g(builder);
+  builder.setInsertionPoint(target);
+  SmallVector<int64_t> indices = llvm::to_vector(getIndices());
+  auto numOperands = target->getNumOperands();
```

Comment:
nit: please spell out the type.
(the rule we follow is here: https://llvm.org/docs/CodingStandards.html#use-auto-type-deduction-to-make-code-more-readable)

Reply:
sure will change